src/core_plugins/kibana/public/visualize/visualize_constants.js

```diff
@@ -0,0 +1,7 @@
+export const VisualizeConstants = {
+  LANDING_PAGE_URL: '/visualize',
```

Comment:
The way these `*_URL` variables are used implies that `*_PATHNAME` would be a more appropriate suffix (The code assumes they will not have query strings, hosts, ports, etc.).